code-challenges.test.js

```diff
@@ -33,9 +35,33 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+describe("divByThree", () => {
+    // "divByThree" is the string arguement for the describe function
+    it("decides if the number inside it is evenly divisible by three or not",
+    () => {
+    // the "it" statement explains that the function should tell us whether the numbers within the variables are divisible by 3
+        expect(divByThree(object1.number)).toEqual("15 is divisible by three")
+        expect(divByThree(object2.number)).toEqual("0 is divisible by three")
+        expect(divByThree(object3.number)).toEqual("-7 is not divisible by three")
+    // three expect statements are used to call the function for each variable's number and compares the output
+    })
```

Comment:
Pay attention to what the prompt says the function should take as an argument. It should be an object, but here, you are passing in a number.